src/styles/Input.scss

```diff
@@ -38,9 +39,40 @@
             border: 1px ;
             outline-style: solid;
         }
-
     }
+}
+
+.clr-subtext{
+    display: block;
+    font-size: 12px;
+    line-height: 2rem;
+    margin-top: .3rem;
+    fill: #666666;
+    font-weight: 400;
+    letter-spacing: 0px;
+    margin-top: 1.5rem;
+    margin-left: -16.4rem;
+}
+
+.clr-error .clr-subtext{
+    color: $ERROR;
+    font-weight: 400;
+    font-size: 12px;
+    letter-spacing: 0px;
+    line-height: 20px;
+    margin-top: 2rem;
+    margin-left: -17.5rem;
+}
 
+.clr-validate-icon {
+    color: $ERROR-ICON;
+    margin-top: 0.4rem;;
+}
+
+.clr-error{
```

Comment:
Can we format this file in SCSS standards ? clr-error and clr-subtext classes are defined more than once

Reply:
clr-error is now defined only once and clr-subtext is defined twice, one for normal helperText and another for errorHelperText.